### PR TITLE
New Security and Privacy Sections

### PR DESCRIPTION
**Pull request by espinr**
First version of the security and privacy sections, very generic. 
Still to decide if we define a concrete CSP (limiting the external domains, blocking dynamic scripts, etc.).
I'll follow the conversations in #46, and I'll update the document.

***
<a href="https://pr-preview.s3.amazonaws.com/espinr/miniapp-packaging/pull/47.html" title="Last updated on Feb 25, 2022, 3:41 PM UTC (14218c8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-packaging/47/b50a97e...espinr:14218c8.html" title="Last updated on Feb 25, 2022, 3:41 PM UTC (14218c8)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -641,26 +641,69 @@ <h2>Accessibility Considerations</h2>
         </p>
     </section>   
 
-    <section id="sec-security-privacy">
-        <h2>Security and Privacy Considerations</h2>
+    <section id="sec-security">
+        <h2>Security Considerations</h2>
+        <p>
+            This specification defines a mechanism to pack, distribute and load MiniApps through a concrete configuration that specifies the app's entry point and different routes (i.e., the manifest's <a href="https://www.w3.org/TR/miniapp-manifest/#pages-member"  data-link-type="dfn">`pages`</a> member). [=User agents=] SHOULD mitigate the potential vulnerability during the initialization and loading phase by limiting the type of resources to be loaded and implementing a sandboxed environment that blocks access to sensitive features by default.
```
Comment (xfq) [inline index.html:647]: What does "sensitive features" mean here? Does it mean something like [powerful features](https://www.w3.org/TR/permissions/#powerful-features)? Or something else?

File: index.html [hunk 2]
```diff
@@ -641,26 +641,69 @@ <h2>Accessibility Considerations</h2>
...
-            <p>No specific digital signature mechanism, cryptographic technology, or algorithm is recommended in this specification. MiniApp vendors MAY or MAY NOT implement digital signature mechanisms, according to their use cases. When implemented, the <a href="#sec-miniapp-digital-signature-requirements"></a> shall be followed.</p>
-
-            <p>Some examples of technologies and encryption methods are introduced in the <a href="#sec-signature-schemes"></a>.</p>
-
-        </section>
-        <section>
-            <h3>Content Confidentiality</h3>
-            <p>This specification does not recommend any encryption mechanism for the [=MiniApp package=] to protect its confidentiality. However, it doesn't preclude an implementation from deploying some encryption mechanism for special purposes.</p>
+            <p>
+                No specific [=digital signature=] mechanism, cryptographic technology, or algorithm is recommended in this specification. MiniApp vendors MAY or MAY NOT implement digital signature mechanisms, according to their use cases. When implemented, the <a href="#sec-miniapp-digital-signature-requirements"></a> shall be followed.
```
Comment (xfq) [inline index.html:681]: some nits:

* I think the RFC 2119 MAY already means "MAY or MAY NOT"
* "shall" is also an RFC 2119 keyword, so you may (no pun intended) want to make it all capitals

File: index.html (hunk 1 above)
Comment (espinr) [reply index.html]: Thanks @xfq for the reference. Indeed, powerful feature is the definition I needed. Changed in e7f771eb701bfa85cee18ea3b673f1e80912e433

File: index.html (hunk 2 above)
Comment (espinr) [reply index.html]: Well spotted. Thanks @xfq , solved with 14218c8228693ff9b62f762fcf3f643f20b70ad0

**Pull request merged**